02.calendar/calendar.js

```diff
@@ -0,0 +1,33 @@
+import { DateTime } from "luxon";
+import minimist from "minimist";
+
+const argv = minimist(process.argv.slice(2));
+const today = DateTime.now();
+
+/*入力の有無によって値を変更*/
+const entered_year = parseInt(argv.y) || today.year;
```

Comment:
```suggestion
const enteredYear = parseInt(argv.y) || today.year;
```

JSはキャメルケースで書くことが多いです。

そもそもの話として、entered以外のyearがあるわけではないので、yearだけでも良いかもしれません。

Reply:
おっしゃる通りです。
ご指摘の通り、変数名`year`と`month`に変更しました。
[🐛 fix: 設定の年月の変数名変更](https://github.com/reckyy/js-practices/pull/2/commits/43533fbd7e44ea2f9b8c30c6e6c2de035b17b419)